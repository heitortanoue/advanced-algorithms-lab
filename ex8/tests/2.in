1234 5
54545349 1
1234456 3
