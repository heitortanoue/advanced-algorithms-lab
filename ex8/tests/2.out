4321
94545345
6544321
